stages/org.osbuild.kickstart

```diff
@@ -568,6 +633,9 @@ def main(tree, options):  # pylint: disable=too-many-branches
 
     config += make_groups(options.get("groups", {}))
     config += make_users(options.get("users", {}))
+    rootpw_command = make_rootpw(options)
```

Comment:
Another style choice :) I'd follow the style of make_groups and make_users, passing in only the needed dict and returning the list, empty or not. This also makes it easier to test without a full options dict.

Reply:
related change has been done in this commit https://github.com/osbuild/osbuild/pull/1560/commits/903b5622b703aa8d9a77acc4ce8a814cf2dd877c